include/alpaka/vec/Vec.hpp

```diff
@@ -988,7 +1014,9 @@ namespace alpaka
             using IdxOffset = std::integral_constant<std::intmax_t, static_cast<std::intmax_t>(dim::Dim<TExtent>::value) - static_cast<std::intmax_t>(TDim::value)>;
             return
 #ifdef ALPAKA_CREATE_VEC_IN_CLASS
-            Vec<TDim, size::Size<TExtent>>::template
+                Vec<TDim, size::Size<TExtent>>::template
```

Comment:
This would be my preferred solution but this will require changes to existing code using alpaka.